packages/react-storage/src/components/StorageBrowser/Views/LocationDetailView/Controls.tsx

```diff
@@ -30,6 +30,7 @@ export const LocationDetailViewControls: LocationDetailViewControls = () => {
     <>
       <Navigate />
       <Title />
+      <Refresh />
```

Comment:
at the moment, the `Refresh` button has the same grid placement with the `ActionSelect. They should be on the same row, but I think for this specific location detail view, we need to have `Refresh` have `grid-column: 26 / -2` instead. How should we conditionally set this since in the locations view, `Refresh` should have `grid-column: 26 / -1`